### DISCUSSION TITLE
Как убрать время публикации. Как убрать плашку "Вложения". И ещё немного вопрсоов.

### DISCUSSION THREAD

**Discussion by Lenivi** [MrZillaGold/VK2Discord#206] [Q&A] [answered]
Начну с первого вопроса. Как убрать время публикации?
![image](https://user-images.githubusercontent.com/86191624/122677251-007b2900-d1fb-11eb-87f2-b2eaaa30e496.png)
Второй вопрос. Как убрать плашку "Вложения"?
![image](https://user-images.githubusercontent.com/86191624/122677270-17ba1680-d1fb-11eb-8063-3c0dddc9b05a.png)
(Я не имею в виду убрать вообще вложения, а сделать примерно вот так)
![image](https://user-images.githubusercontent.com/86191624/122677293-2dc7d700-d1fb-11eb-89f3-007c52ce9057.png)
Третий и пока последний вопрос. Как вместо названия группы, сделать гиперссылку на конкретный пост?
![image](https://user-images.githubusercontent.com/86191624/122677315-4a640f00-d1fb-11eb-989a-71628c4a700e.png)
(Вот пример)
![image](https://user-images.githubusercontent.com/86191624/122677334-5ea80c00-d1fb-11eb-8aa2-9bdb50a0521b.png)

P.S Прошу прощения что до этого задал вопрос не в том топике, слишком мало разбираюсь в сайте GitHub

**Comment by egorprnn** [comment, marked as answer]
1. Время публикации убрать нельзя, вынесу в конфиг в следующем обновлении.
2. Заголовок "Вложения" тоже нельзя отключить, также вынесу в конфиг.
3. Эта механика была полностью удалена и поддерживаться не будет, заголовок кликабельный и через него можно перейти на запись.

**Reply by Lenivi** [reply]
> заголовок кликабельный и через него можно перейти на запись.

Но, можно ли убрать аватарку и имя группы и вместо неё сделать свою кликабельную надпись?

**Reply by egorprnn** [reply]
> Эта механика была полностью удалена и поддерживаться не будет

https://github.com/MrZillaGold/VK2Discord/releases/tag/1.9.0

**Reply by Lenivi** [reply]
> > Эта механика была полностью удалена и поддерживаться не будет
> 
> https://github.com/MrZillaGold/VK2Discord/releases/tag/1.9.0

Понял, спасибо за ответы

**Reply by Lenivi** [reply]
Ох уж, я вас наверное заколебал вопросами. Как бы я не пытался, но у меня не получается чтобы под плашкой "Вложения", было вложение с изображением. Как в общем то говоря, сделать так чтобы под этой плашкой, было вложение изображения?
![image](https://user-images.githubusercontent.com/86191624/122681479-ed258900-d20d-11eb-85a4-17294b5e7ea1.png)
